Source/Core/DolphinWX/Debugger/CodeView.cpp

```diff
@@ -366,6 +415,8 @@ void CCodeView::OnMouseUpR(wxMouseEvent& event)
   menu.AppendSeparator();
 #endif
   menu.Append(IDM_RENAMESYMBOL, _("Rename &symbol"))->Enable(isSymbol);
+  menu.Append(IDM_SETSYMBOLSIZE, _("&Set symbol size"))->Enable(isSymbol);
```

Comment:
Is it an issue since we can switch between accelerators by pressing the key again? It doesn't seem so on Windows. Furthermore, there are also duplicate accelerators for "a", do you want me to fix them as well?